defaults/main.yml

```diff
@@ -8,3 +8,7 @@ etcd_certs_root: "/etc/ssl/etcd"
 kube_auth_dir: "/etc/kubernetes/auth"
 kube_users_map: {}
 kube_policies_list: []
+kube_scheduler_policy_config_file_src: ""
+kube_scheduler_policy_config_dir: ""
+kube_scheduler_policy_config_file: ""
```

Comment:
call this has file_name. so that var clearly identifies it as the name of the file and not the file path.